media/redesign/stylus/submission.styl

```diff
@@ -122,3 +113,49 @@
         }
     }
 }
+
+.field-note {
+    color: #666;
+    font-family: inherit;
+    set-smaller-font-size();
+    font-style: normal;
+    font-weight: normal;
+
+    p& {
+        margin: ($grid-spacing / 4) 0;
+    }
+
+    .wordcount & {
+        float: right;
+    }
+}
+
+.field-management {
+    bidi-value(float, right, left);
+    margin: 0 0 5px;
+}
+
+.field-account {
+    clear: both;
+}
+
+#field-beta {
+    @extend $checkbox-label-container;
+}
+
+#profiles {
+    label {
+        margin: 0 0 5px;
+        bidi-value(float, left, right);
+        bidi-style(margin-right, 10px, margin-left, 10px);
+    }
+}
+
+.key {
+    label {
+        display: block;
+    }
+}
+
+
+
```

Comment:
Too bad my editor is better at adding an extra space at the end of files than it is at removing them ;)